src/components/app.js

```diff
@@ -21,12 +21,19 @@ export default React.createClass({
     this.setState({user});
   },
   render() {
-    //return <Circle user={this.state.user} />;
+    return <Circle user={this.state.user} onPress={ () => {
+        if (!this.currentUser.ref) return <View />;
```

Comment:
Actually, this does need to be handled here (you might want to show the teams for another user). So this should be:

```
this.currentUser.getRef().then(userRef => {
  this.props.navigator.push({
    ...
    passProps: {
      userRef,
      ...
```